easybuild/easyconfigs/p/Perl/Perl-5.24.1-GCCcore-6.3.0.eb

```diff
@@ -1126,6 +1126,10 @@ exts_list = [
         'source_urls': ['http://search.cpan.org/CPAN/authors/id/J/JE/JETTERO/'],
         'checksums': ['6855ce5615fd3e1af4cfc451a9bf44ff29a3140b4e7130034f1f0af2511a94fb'],
     }),
+    ('Time::HiRes', '01.02', {
+        'source_tmpl': 'Time-HiRes-01.02.tar.gz',
+        'source_urls': ['http://search.cpan.org/CPAN/authors/id/D/DE/DEWEG/'],
```

Comment:
@SethosII For recent `Perl` easyconfigs (5.24.1 and more recent) we also include checksums for extensions, please do this for `Time::HiRes` too?

The easiest way should be via `eb --inject-checksums --force ...` (you need `--force` since there are existing checksums).

Reply:
@boegel I changed this.